content/learn/books/adopting-innersource-principles-and-case-studies.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: page
```

Comment:
As I understand (but I can mistake) since this page is a [Leaf Bundle](https://gohugo.io/content-management/page-bundles/) there is no need to specify `layout: page`.

Reply:
You are right. Works equally well without that.
I removed it from all books pages.